Add repositories and pluginRepositories to pom.xml

rvanheest opened the pull request:
@wilkos-dans and I came to this after it did not work for him. @janvanmansum, you forgot to add that this morning!

rvanheest committed:
Add repositories and pluginRepositories to pom.xml

rvanheest commented:
To be honest, I don't think we should actually add the Dans dependencies. This also requires to add the mirror to the `.m2/settings.xml`

rvanheest commented:
Could you also add this project to Travis to avoid problems like this in the future?

janvanmansum commented on pom.xml:
```diff
@@ -86,6 +86,24 @@
             <version>1.1.1</version>
         </dependency>
     </dependencies>
+    <repositories>
```
Good point, only it should be:
```
 <repositories>
        <repository>
            <id>DANS</id>
            <releases>
                <enabled>true</enabled>
            </releases>
            <url>http://maven.dans.knaw.nl/</url>
        </repository>
    </repositories>
    <pluginRepositories>
        <pluginRepository>
            <id>DANS</id>
            <releases>
                <enabled>true</enabled>
            </releases>
            <url>http://maven.dans.knaw.nl/</url>
        </pluginRepository>
    </pluginRepositories>
```
Those are accessible from outside the DANS network.

janvanmansum replied:
While you are at it you might also add the travis config.

rvanheest commented:
Closing this PR for now. Moved these changes to the other PR at #21.
@janvanmansum I added the travis link in #21, but I think you have to add it to travis yourself. I don't have those rights, if I remember correctly.
